app/src/controllers/user/actionCreators.js

```diff
@@ -68,3 +70,40 @@ export const setStartTimeFormatAction = (format) => ({
   type: SET_START_TIME_FORMAT,
   payload: format,
 });
+
+export const assignToProjectAction = (project, { userId, userRole }) => (dispatch) => {
+  const { projectName, entryType } = project;
+  const data = {
+    userNames: {
+      [userId]: userRole,
+    },
+  };
+  return fetch(URLS.userInviteInternal(projectName), {
+    method: 'put',
+    data,
+  }).then(() => {
+    dispatch({
```

Comment:
as far as I know we use sagas for async actions, not thunks, right @DiscoElevator ?

Reply:
Event though we have thunks here, we plan to move to sagas, so I think we should start using them here at least for new actions.